packages/terra-consumer-nav/src/components/nav-help/NavHelpPopup.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import PopupHeights from 'terra-popup/lib/_PopupHeights';
+import PopupHeights from 'terra-popup/lib/PopupHeights';
```

Comment:
whats up with this change? did you accidentally upgrade terra pop up to the new version
https://github.com/cerner/terra-core/pull/912/files#diff-c06cdb21632b409e03d17daa59d1b72aR2
?

Reply:
This change was necessary to fix lint errors in the Travis build.  The package.json file for terra-consumer-nav specifies a terra-popup version on ^1.8.0, which allows npm install to pull in the latest version 1.12.0.

Reply:
@chris-bert what lint error did it fix? this was like this in master already and the build wasn't failing right? 

There is an issue in 1.12 @zhongyn has been looking into, but it seems we only encounter this if we pass a height of auto.

The deployed site of this branch looks fine

Reply:
We were getting the following lint errors because the 1.12.0 version of terra-pop got installed and in that version the name of the _PopupHeights.js file was changed to PopupHeights.js.  Not sure why this didn't cause a problem for the master build.

/home/travis/build/cerner/terra-consumer/packages/terra-consumer-nav/src/components/nav-help/NavHelpPopup.jsx
  3:26  error  Unable to resolve path to module 'terra-popup/lib/_PopupHeights'  import/no-unresolved
  3:26  error  Missing file extension for "terra-popup/lib/_PopupHeights"        import/extensions
/home/travis/build/cerner/terra-consumer/packages/terra-consumer-nav/src/Nav.jsx
  7:26  error  Unable to resolve path to module 'terra-popup/lib/_PopupHeights'  import/no-unresolved
  7:26  error  Missing file extension for "terra-popup/lib/_PopupHeights"        import/extensions

Reply:
ahh, its because we have a lerna repo, so we don't utilize lock files. This new version was just released. This doesn't plague our apps because they use lock files, but this is an issue in this project now.
Thanks